traveltime-spec.yml

```diff
@@ -458,6 +477,11 @@ components:
             - public_transport
             - driving
             - driving+public_transport
+            - driving+ferry
+            - cycling
```

Comment:
Do we have simply cycling (without ferry)? The same for walking and driving.

Reply:
I just followed the description from /v4/time-filter/fast

![image](https://github.com/traveltime-dev/traveltime-openapi-specification/assets/126453011/760fca73-2c90-49a6-8153-e065d38e9c34)

Reply:
I think this is out of date, we won't be supporting cycling/driving/walking due to costs for now.

Reply:
@MockusTravelTime - not sure who to ping about updating the docs for time-filter/fast and proto with the above

Reply:
please send supported Transport types s for fast/proto endpoints to web dev or DM me whoever knows them